ci/docker/wasm32-wasi/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rust:1-slim
+
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
+  ca-certificates \
+  curl wget \
+  software-properties-common lsb-release gpg-agent gnupg gnupg1 gnupg2 \
+  make \
+  perl \
+  gcc \
+  xz-utils \
+  libc6-dev
+
+RUN bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)" -- 14
+RUN update-alternatives --install /usr/bin/llvm-config llvm-config /usr/bin/llvm-config-14 14
+RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-14 14
```

Comment:
Is this needed since wasi-sdk has its own version of clang?

Reply:
That's probably an old try of someting else altogether. I'll remove it.